### ISSUE TITLE
Escape function before connection

### ISSUE THREAD

**Issue by wviveiro** [jeremydaly/serverless-mysql#28]
Hi there!

I am creating a small library to help me building my mysql queries. The goal is to build the query through series of functions and then be able to output the query ready. Once it is ready, I execute **query(mystring)**. However I noticed that I can't run **mysql.getClient()** before running **mysql.connect()** and therefore, I can't get the escape function from **mysql.getClient().escape**

TLDR

Can you please export the function *escape* from *require('mysql')*? On this way I don't have to *mysql.connect()* and *mysql.getClient()* to use the function.

Thanks!

**Comment by jeremydaly** [comment]
Hi @wviveiro,

Thanks for the suggestion. I will investigate and add to the roadmap.

- Jeremy

**Comment by sebastientromp** [comment]
Hey! I would be interested in this as well :)

**Comment by jeremydaly** [comment]
Resolved with @wviveiro's patch.
